userapi/src/main/java/fk/prof/userapi/ServiceUnavailableException.java

```diff
@@ -0,0 +1,15 @@
+package fk.prof.userapi;
+
+/**
+ * Created by gaurav.ashok on 26/06/17.
+ */
+public class ServiceUnavailableException extends Exception {
```

Comment:
Class hanging in middle of nowhere. Put this in http package

Reply:
Moved to http package.